cores/esp8266/Updater.cpp

```diff
@@ -62,6 +62,11 @@ void UpdaterClass::_reset() {
 }
 
 bool UpdaterClass::begin(size_t size, int command, int ledPin, uint8_t ledOn) {
+#if defined(ERASE_CONFIG_H) && !defined(HOST_MOCK)
```

Comment:
ERASE_CONFIG_H always defined, drop the condition (leaving host_mock)

Reply:
okay